packages/financial-templates-lib/test/truffle-fixture.js

```diff
@@ -0,0 +1,2 @@
+// Export the truffle-fixture file from `/core/test` so that we can use the `.deployed()` syntax in our tests.
```

Comment:
where are you using this? are you using this in other packages? IMHO this is something we should strictly _not_ do. This somewhat defeats a lot of the gains we get from using hardhat. You dont want to re-run all the migrations for a test and rather deploy what you need in the tests. I think this should be the pattern that we follow for all packages that are _not_ `core`. This is how other packages are set up right now. Liquidator/disputer/monitor dont rely on any truffle migrations for this exact reason.

Reply:
Looking further down, I see you are doing things like
```
timer = await Timer.deployed();
perpFactory = await PerpetualCreator.deployed();
```

IMHO you should refactor this to _not_ be dependent on the core migrations. this will drastically speed up your unit tests and follow the patterns in the other non-core packages.

Reply:
@chrismaree I need this because the `EMP/PerpCreator` contracts can't be deployed unless I link their their EMPLib/PerpLib contracts which I can't seem to figure out how to do in hardhat

Reply:
Ended up fixing this by adding `await PerpetualCreator.link(await PerpetualLib.new())`